datahub-web-react/src/app/entity/shared/types.ts

```diff
@@ -52,6 +53,8 @@ export type GenericEntityProperties = {
     /** Dataset specific- TODO, migrate these out */
     editableSchemaMetadata?: Maybe<EditableSchemaMetadata>;
     editableProperties?: Maybe<DatasetEditableProperties>;
+    /** Dashboard specific- TODO, migrate these out */
+    charts?: Maybe<EntityRelationshipsResult>;
```

Comment:
fwiw, a cleaner approach would be to just call useBaseEntity() from the DashboardChartsTab

Reply:
actually i do use that... this is unused and can be removed!